src/components/thumbnail-carousel.ts

```diff
@@ -109,25 +109,50 @@ export class FrigateCardThumbnailCarousel extends FrigateCardCarousel {
       return;
     }
 
-    return html`<div
-      class="embla__slide"
-      @click=${(ev) => {
-        if (this._carousel && this._carousel.clickAllowed()) {
-          dispatchFrigateCardEvent<ThumbnailCarouselTap>(this, 'carousel:tap', {
-            slideIndex: slideIndex,
-            target: parent,
-            childIndex: childIndex,
-          });
-        }
-        stopEventFromActivatingCardWideActions(ev);
-      }}
-    >
-      <img
-        aria-label="${mediaToRender.title}"
-        src="${mediaToRender.thumbnail}"
-        title="${mediaToRender.title}"
-      />
-    </div>`;
+    if (mediaToRender.frigate?.event?.retain_indefinitely == true) {
+      return html`<div
+        class="embla__slide"
+        @click=${(ev) => {
+          if (this._carousel && this._carousel.clickAllowed()) {
+            dispatchFrigateCardEvent<ThumbnailCarouselTap>(this, 'carousel:tap', {
+              slideIndex: slideIndex,
+              target: parent,
+              childIndex: childIndex,
+            });
+          }
+          stopEventFromActivatingCardWideActions(ev);
+        }}
+      >
+        <img
+          aria-label="${mediaToRender.title}"
+          src="${mediaToRender.thumbnail}"
+          title="${mediaToRender.title}"
+        />
+        <img
```

Comment:
(this `img` is the only different part, so thiere is where the terniary operator would come in as suggested above)

Reply:
Gotcha, makes sense.